tools/check_yaml

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash -e
+
+success=1
+YAMLS=$(git ls-files schedule/ test_data/ | grep '.ya\?ml$' | xargs echo)
+
+if test -n "$YAMLS"; then \
+        export PERL5LIB=${PERL5LIB_} ; echo "$YAMLS" | xargs tools/test_yaml_valid | grep -Ev "has valid yaml syntax|has valid schema";\
+        else \
+        echo "No yamls modified.";\
+fi
+if test -n "$YAMLS"; then \
```

Comment:
Isn't this check done twice and can be merged? :thinking: 

Line 6 and 11 look exactly the same to me and I don't see any changes to `$YAMLS` itself.

Reply:
You are right, let me revise my code